rules-tests/CodingStyle/Rector/ClassMethod/DataProviderArrayItemsNewlinedRector/Fixture/some_class.php.inc

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Rector\Tests\CodingStyle\Rector\ClassMethod\DataProviderArrayItemsNewlinedRector\Fixture;
+
+use PHPUnit\Framework\TestCase;
+
+final class ImageBinaryTest extends TestCase
+{
+    /**
+     * @dataProvider provideData()
+     */
+    public function testGetBytesSize(string $content, int $number): void
+    {
+        // ...
+    }
+
+    public function provideData(): array
+    {
+        return [['content', 8], ['content123', 11]];
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\CodingStyle\Rector\ClassMethod\DataProviderArrayItemsNewlinedRector\Fixture;
+
+use PHPUnit\Framework\TestCase;
+
+final class ImageBinaryTest extends TestCase
+{
+    /**
+     * @dataProvider provideData()
+     */
+    public function testGetBytesSize(string $content, int $number): void
+    {
+        // ...
+    }
+
+    public function provideData(): array
+    {
+        return [
```

Comment:
Maybe turn them into one `yield` per array item instead?

Reply:
There another rule for that already :) 
This rule is mainly for Test Gen AI styling.